src/v2_5/data.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2018 The openrtb-rust authors
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use serde_utils;
+use super::segment::Segment;
+
+// This object used by publishers to pass additional attributes about the user or content.
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub struct Data {
+    // The data extension object that contains community extensions.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    ext: Option<serde_utils::Ext>,

```

Comment:
not an issue but it`s prettier to add the `ext` field as  last

Reply:
as specs